build-mac.sh

```diff
@@ -4,6 +4,11 @@
 QT_BREW_PATH=$(brew --prefix qt@6)
 CMAKE_FLAGS="-DQT_DIR=${QT_BREW_PATH}/lib/cmake/Qt6 -DENABLE_NOGUI=false"
 
+# For some reason the system xxhash library doesn't get properly linked,
+# at least on my M1. The clang command gets -lxxhash, but probably needs
+# -L/opt/homebrew/Cellar/xxhash/... to actually find the library.
+CMAKE_FLAGS+=" -DUSE_SYSTEM_XXHASH=OFF"
```

Comment:
```suggestion
if [[ $(arch) == 'arm64' ]]; then
  CMAKE_FLAGS+=" -DUSE_SYSTEM_XXHASH=OFF"
fi
```
please gate this being an arm64 check like above. i see no reason to change what our CI does at this time

Reply:
done